src/community/pom.xml

```diff
@@ -98,7 +98,7 @@
             <descriptor>release/ext-imagemosaic-jdbc.xml</descriptor>
             <descriptor>release/ext-smart-data-loader.xml</descriptor>
             <descriptor>release/ext-cov-json.xml</descriptor>
-            <descriptor>release/ext-schemaless-features.xml</descriptor>
+            <descriptor>release/ext-schemaless-mongo.xml</descriptor>
```

Comment:
I see a change on the release artifact name, there is documentation where this could be updated?

Reply:
fixed thanks!